include/ace/managers/game.h

```diff
@@ -26,40 +24,60 @@ typedef struct _tGameState {
 	tGameCb cbCreate;
 	tGameCb cbLoop;
 	tGameCb cbDestroy;
-	struct _tGameState *pPrev;
+	tGameCb cbSuspend;
+	tGameCb cbResume;
+	struct _tGameState *pStatePrev;
 } tGameState;
 
 typedef struct _tGameManager {
-	UBYTE ubStateCount;
-	UBYTE isRunning;
-	tGameState *pStateFirst;
+	tGameState *pState;
 } tGameManager;
 
-/* Globals */
-
-extern tGameManager g_sGameManager;
-
 /* Functions */
 
-void gameCreate(void); /* First gameState needs to be added by gamePushState after calling this function */
-
-void gameDestroy(void);
+void gameExit();
 
 UBYTE gameIsRunning(void);
 
-void gamePushState(tGameCb cbCreate, tGameCb cbLoop, tGameCb cbDestroy);
-
-void gamePopState(void);
-
-void gameChangeState(tGameCb cbCreate, tGameCb cbLoop, tGameCb cbDestroy);
-
-void gameChangeLoop(tGameCb cbLoop);
-
-void gameProcess(void);
-
-void gameClose(void);
-
-void gameKill(char *szError);
+tGameManager *gameManagerCreate(void);
+
+void gameManagerDestroy(
+	tGameManager *pGameManager
+);
```

Comment:
wtf with this formatting? it fits in single line, why each param in one line? makes the header way too big.